react-app/src/mgWebGL.js

```diff
@@ -883,6 +883,14 @@ var icosaIndices = icosaIndices2;
 
 var icosaNormals = icosaVertices;
 
+function isDarkBackground(r, g, b) {
+    const brightness = r * 0.299 + g * 0.587 + b * 0.114
+    if (brightness >= 0.5) {
+        return false
+    }
+    return true
+}
```

Comment:
@stuartjamesmcnicholas I have refactored your code into a separate function, which is used by `mgWebGL` to determine the colour of the atom labels and by `BabyGruMoleculeCard` to determine whether the background is "dark" or not.